### ISSUE TITLE
Issued while checking for update of Fennec F-Droid, Report 1

### ISSUE THREAD

**Issue by Universalizer** [Tobi823/ffupdater#260]
### **Thanks for your app, but issued while checking for update of Fennec F-Droid, Report 1 as follows :-**

Stacktrace:
```
java.lang.Exception: The app does not support the device
	at de.marmaro.krt.ffupdater.device.DeviceAbiExtractor.findBestAbiForDeviceAndApp(Unknown Source:41)
	at de.marmaro.krt.ffupdater.app.impl.FennecFdroid.findLatestUpdate$ffupdater_release(SourceFile:64)
	at de.marmaro.krt.ffupdater.app.impl.AppBase.findAppUpdateStatus(Unknown Source:64)
	at de.marmaro.krt.ffupdater.app.impl.AppBase.access$findAppUpdateStatus(Unknown Source:0)
	at de.marmaro.krt.ffupdater.app.impl.AppBase$checkForUpdateAsync$2$1.invokeSuspend(Unknown Source:115)
	at w3.a.resumeWith(Unknown Source:8)
	at j4.i0.run(Unknown Source:100)
	at m4.d.run(Unknown Source:11)
	at n4.j.run(Unknown Source:2)
	at n4.a$a.run(SourceFile:82)
```
Device information:
| Key | Value |
| --- | --- |
| FFUpdater version | 77.3.1 (115) release |
| Device | SM-J600G (j6ltedx, j6lte, exynos7870) |
| Manufacturer | samsung (samsung) | 
| Supported ABIs | armeabi-v7a, armeabi |
| Android version | 10 (SDK: 29) |
| OS | Android 32-bit Only (Not 64-bit)  |         

_By the way can you, “add user feedback within your app menu”, so that direct reporting can be possible._

**Comment by Tobi823** [comment]
I guess I found the bug

**Comment by Tobi823** [comment]
I will release the fixed version (77.3.4) in the next two hours
